src/app/email-validator/email.validator.ts

```diff
@@ -1,17 +1,16 @@
-import {Control} from '@angular/common';
-interface IValidationResult {
-  [key: string]: boolean;
-}
+import {FormControl} from "@angular/forms";
 
 export class EmailValidator {
-  static validEmail(control: Control): IValidationResult {
```

Comment:
Because of the upgrade to @angular/forms v2.0.0. RC6, this does not fly anymore. Or, maybe I am mistaken and we indeed need to re-add it